packages/syntax-core/src/SelectList/SelectList.test.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable testing-library/no-await-sync-events */
+import { render, screen } from "@testing-library/react";
+import SelectList from "./SelectList";
+import { expect, vi } from "vitest";
+import userEvent from "@testing-library/user-event";
+
+const SelectDropdown = ({
+  numberOfOptions,
+  onChange,
+}: {
+  numberOfOptions: number;
+  onChange: () => void;
+}) => {
+  const options = [];
+  for (let i = 1; i <= numberOfOptions; i++) {
+    options.push({ value: String(i), label: String(i) });
+  }
+  return (
+    <SelectList
+      selectedValue=""
+      onChange={onChange}
+      placeholderText="placeholder"
+      label="Select"
+    >
+      <>
+        {options.map((o) => (
+          <SelectList.Option key={o.value} value={o.value} label={o.value} />
```

Comment:
`label`, should we `label={o.label}` instead of `label={o.value}`. Also, you can destructure the arguments:

```jsx
{options.map(({value, label}) => (
  <SelectList.Option key={value} value={value} label={label} />
))}